readthedocs/doc_builder/base.py

```diff
@@ -48,6 +45,7 @@ def __init__(self, build_env, python_env, force=False):
         self.project = build_env.project
         self.config = python_env.config if python_env else None
         self._force = force
+        self.cwd = self.project.checkout_path(self.version.slug)
```

Comment:
Don't you think it's better to call this variable `checkout_path` (or `project_checkout_path` or `project_path`) that communicates better that it points to the project's path instead of just `cwd` that could be anywhere?